build/BuildSteps.cs

```diff
@@ -107,17 +107,14 @@ public static void AddDistLib()
 
         public static void AddPythonWorker()
         {
-            var pythonWorkerPath = Path.Combine(Settings.OutputDir, "python-worker");
-            Directory.CreateDirectory(pythonWorkerPath);
-            using (var client = new WebClient())
-            {
-                client.DownloadFile(Settings.PythonWorkerUrl, Path.Combine(pythonWorkerPath, "worker.py"));
-                client.DownloadFile(Settings.PythonWorkerConfigUrl, Path.Combine(pythonWorkerPath, "worker.config.json"));
-            }
-
             foreach (var runtime in Settings.TargetRuntimes)
             {
-                FileHelpers.RecursiveCopy(pythonWorkerPath, Path.Combine(Settings.OutputDir, runtime, "workers", "python"));
+                // Python worker's dependencies are platform dependent and need to be copied accordingly
+                var pythonDir = Path.Combine(Settings.OutputDir, runtime, "workers", "python");
+                var allOsDirectories = Directory.GetDirectories(pythonDir);
+                var pythonPlatformSpecificWorker = Path.Combine(pythonDir, Settings.RuntimesToOS[runtime]);
+                FileHelpers.RecursiveCopy(pythonPlatformSpecificWorker, pythonDir);
+                allOsDirectories.ToList().ForEach(dir => Directory.Delete(dir, recursive: true));
```

Comment:
did you mean to have these 2 in the reverse order? It seems that the last line will just delete all the workers including the one you just copied.

Reply:
Actually I think I may have misunderstood the flow here. The `Windows, Linus, Mac` folders come from the nuget package, right? then you copy all the content of the right one outside it's platform specific folder, then delete all the platform specific folder?

Reply:
@ahmelsayed, actually I meant it the way I have. The directory `workers/python` has 3 runtime directories (`Linux`, `Mac` and `Windows`). I am copying everything from `workers/python/<runtime>/` to `workers/python/` and then deleting all the three runtime directories.

So, I end up with `workers/python/deps`, `workers/python/worker.py` and `workers/python/worker.config.json`.

Reply:
@ahmelsayed sorry, didn't see your reply. Yes, you are right. That's what I am doing.